pkg/function/delete_rds_snapshot.go

```diff
@@ -55,22 +55,22 @@ func (*deleteRDSSnapshotFunc) Name() string {
 	return DeleteRDSSnapshotFuncName
 }
 
-func deleteRDSSnapshot(ctx context.Context, snapshotID string, profile *param.Profile, dbEngine RDSDBEngine) (map[string]interface{}, error) {
```

Comment:
Should this be part of this PR?

Reply:
This fixes an `unparam` linter warning.

Reply:
@KastenMike I am assuming this was flagged by a linter as part of the upgrade. @hairyhum can confirm.

If that's the case, there are 2 options:
- Put all the changes in a single PR, like this one; or
- 2 PRs in this order: the first one addressing the linter warnings, and the second one upgrading the linter version.

Reply:
Upgrading linter version is a github action, which I've already run, so it's just a matter of getting it to the `master` codebase